src/shared/expectedSourceMembers.ts

```diff
@@ -46,16 +46,16 @@ const isSpecialAuraXml = (filePath?: string): boolean =>
 
 // things that never have SourceMembers
 const excludedKeys = [
-  'AppMenu__Salesforce1',
```

Comment:
I changed the key delimiter.  I think this might help some of the tracking issues with namespaced metadata, though the API drops some namespaces, too and this won't fix that.